sympy/core/tests/test_subs.py

```diff
@@ -622,14 +622,14 @@ def test_mul2():
 
 def test_noncommutative_subs():
     x,y = symbols('x,y', commutative=False)
```

Comment:
no need to add space here...this hundreds of times throughout sympy. There has to be some delimiter between logical symbol names and " " or "," is allowed. Changes in expressions is fine.

+1 as is